web/styles/left_sidebar.css

```diff
@@ -1864,6 +1868,51 @@ li.active-sub-filter {
     align-self: baseline;
 }
 
+.zoomed-new-topic {
+    display: grid;
+    grid-template:
+        "starting-offset starting-anchor-element icon-content-gap row-content ending-offset"
```

Comment:
Not a blocker for this PR, but we're starting to stack up some minimally different `grid-template` values here and elsewhere in `left_sidebar.css`. Ideally we should have as few of these as possible, and just adjust `grid-template-columns` or `grid-template-rows` as necessary (and just set 0 values for columns/rows that have no such need for a particular named area).

Adding this to my own to-do list as something to audit.